guide/topics/discordjs/firstCommand.md

```diff
@@ -0,0 +1,163 @@
+
+## Making a command
+In the first sentence of creating the event, I said that all commands are prefixed messages. So, let's try and make a very simple command handler.
+```javascript
+client.on("message", (message) => {
+  if(message.author.bot || !message.guild) return;
+  
+  const split = message.content.split(" ");
+  const command = split[0];
+  const args = split.slice(1);
+});
+```
+
+Before going any further, let's quickly discuss this code. Again, we are listening to the `message` event, nothing different here. In the next line, we are creating a variable named `split` which is assigned to the message content *split with spaces*.
+
+Take this example for example:
+
+The message content is: "!ban user reason", a very simple ban command.
+
+We [split](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split) it on each space - effectively creating an array of words, that would look like this `["!ban", "user", "reason"]`.
+
+With that in mind, in the next line we declare `command` to `split[0]`, which is going to be our first element of the array - the command.
+After that, we have args which is `split`, but with first element (our command) sliced, so we are left with only whatever comes _after_ the command, which are going to be from now on called arguments.
+
+So, for example, `user` or `reason` in our command are going to be stored in the args array, so we can access it later.
+
+Ok, moving on with the code!
+```javascript
+client.on("message", (message) => {
+	if(message.author.bot || !message.guild) return;
+	  
+	const split = message.content.split(" ");
+	const command = split[0];
+	const args = split.slice(1);
+	
+	if(command.toLowerCase() === "!ping".toLowerCase()) {
+		message.channel.send("Pong!");
+	}
+});
+```
+
+Now, a very straightforward `if` statement to check if the `command` part is equal to "!ping". Notice how when checking, after both `command` and the literal command string, and sending a response. Very easy.
+
+## Conclusion
+
+This is the basics of how you can make your own commands. You can of course add much more code to your command. Be creative!
+
+This is the final code that we made to add our simple "ping" command and set up code for further commands:
+
+```javascript
+const { Client } = require("discord.js");
+
+const client = new Client();
+
+client.once("ready", () => console.log("I am ready!"));
+client.on("message", (message) => {
+  if(message.author.bot || !message.guild) return;
+
+  const split = message.content.split(" ");
+  const command = split[0];
+  const args = split.slice(1);
+  
+  if(command.toLowerCase() === "!ping".toLowerCase()) {
+    message.channel.send("Pong!");
+    // Add more code to a command here...
+  }
+})
+
+client.login("-- Your Token --");
+```
+
+In the next tutorial we'll learn how to learn [todo next tutorial]
+<!-- TODO : Add a link -->
```

Comment:
Personally, I would remove this for now until there's a solid idea of what additional entries are planned after this one. I feel for these guides, consistency between different entries is paramount, so that's the main factor in my response here.